### PR TITLE
German LS to CAPP Agile Learning

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Translating the page and other stuff. 

- [x] Redirect /learningspaces to /capp-agile-learning
- [x] Change filenames of testimonials
- [x] Update testimonials
- [x] Page translations
- [x] Homepage translations
- [x] Product overview translations
- [x] Nav translations
- [x] Clean up
- [x] Translate hero image
- [x] Translate what is capp agile image (I put in another 1)
- [x] Translate space image
- [x] Link QR to shared space
- [x] Translate QR image
- [x] Translate question image
- [x] Translate demo image

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
Only the images are left 
capp agile learning defacto 1

File: config.rb [hunk 1]
```diff
@@ -72,7 +72,7 @@
 redirect "capp.html", to: "capp-lms.html"
 redirect "capp-lms.html", to: "capp-bilden.html" if root_locale == :de
 redirect "kundenreferenzen.html", to: "kundenstimmen.html"
-redirect "learningspaces.html", to: "capp-agile-learning.html" if root_locale == :nl
+redirect "learningspaces.html", to: "capp-agile-learning.html"
```
Comment (sn3p) [inline config.rb]: Maybe we should consider conditional blocks here:

```rb
case root_locale
when :nl
  redirect ...
when :de
  redirect ...
end
```

Comment (RonaldDijkstra) [inline config.rb]: I like it

**Comment by RonaldDijkstra** [comment]
When the Germans are back I'll talk to them about the client logos on the references part. For now the Dutch will do the trick. 

Ready for review @sn3p

**Pull request merged**